app/models/domain_block.rb

```diff
@@ -12,11 +12,12 @@
 #
 
 class DomainBlock < ApplicationRecord
-  enum severity: [:silence, :suspend]
+  enum severity: [:silence, :suspend, :media_only]
```

Comment:
I suggest naming it "transparent" or "noop" and having it do absolutely nothing, thus not having to worry about functionality overload with the reject_media boolean.